test/pantry-test.js

```diff
@@ -121,21 +121,21 @@ describe('Pantry', () => {
   });
 
   it('should be able to tell if the user has the ingredients required to cook a given recipe', () => {
-    let result = testPantry.determineIfUserHasEnoughIngredients(testRecipeData[0])
-    expect(result).to.be.false;
+    let result = testPantry.findWhichIngredientsAreShort(testRecipeData[0])
```

Comment:
Should we be using the same variable name `result` for our expect statements?
How can we ensure the correct `result` value is being evaluated?

Reply:
This is how I've seen it done before. I think 'result' is just used for developer empathy. It could be anything we like.